### ISSUE TITLE
[TIDAL] Subscription error message unclear

### ISSUE THREAD

**Issue by rijnhard** [jonaski/strawberry#52]
The real error was that my subscription wasn't active, but the error recieved in code alludes to that `Requested quality is not allowed in user's subscription` but the error I got was `No URI handler implemented for "tidal"` which is misleading and unclear, I think you can just propogate the error from tidal

```
13:39:25.736 DEBUG Player:279                       URL handler for QUrl("tidal:17793623") is loading asynchronously
13:39:25.989 ERROR TidalService:904                 Tidal: "Requested quality is not allowed in user's subscription"
13:39:25.989 DEBUG Player:247                       URL handler for QUrl("tidal:17793623") returned QUrl("tidal:17793623")
13:39:25.993 ERROR GstEnginePipeline:575            ErrorMessageReceived 3 "gsturidecodebin.c(1427): gen_source_element (): /GstPlayBin:playbin-30/GstURIDecodeBin:uridecodebin2"
13:39:26.024 ERROR GstEngine:562                    Gstreamer error: 4024 12 "No URI handler implemented for \"tidal\"."
13:39:26.024 ERROR logging:60                       Source ID 873 was not found when attempting to remove it
__logging_message__13:39:26.025 WARN  unknown                          QTimeLine::start: already running
```

**Comment by jonaski** [comment]
I agree. We'll add a mechanism through the url handler signal with an extra variable that contains the error message if any.

**Comment by jonaski** [comment]
Fixed in c91cef3
